libs/components/config/src/lib/params.spec.ts

```diff
@@ -278,6 +278,48 @@ describe('SkyAppRuntimeConfigParams', () => {
     expect(params.getAllKeys()).toEqual([]);
   });
 
+  it('should add provided params to a url link', () => {
+    const params: SkyAppRuntimeConfigParams = new SkyAppRuntimeConfigParams(
```

Comment:
Use type inference here and in the other tests with a similar structure.

Reply:
Sounds good. Prettier did some adjusting so it looks like there are more changes than are necessary; it is what it is.